RELEASE_NOTES.md

```diff
@@ -2,8 +2,18 @@
 
 ## Summary
 
-This is the initial release, extracted from the [SDK v1.0.0rc601](https://github.com/frequenz-floss/frequenz-sdk-python/releases/tag/v1.0.0-rc601).
```

Comment:
Same here, I wouldn't remove this until 1.0 final.

Reply:
Done